core/src/main/java/brooklyn/entity/group/DynamicClusterImpl.java

```diff
@@ -130,12 +130,14 @@ public ZoneFailureDetector apply(final String input) {
 
     private static final Function<Collection<Entity>, Entity> defaultRemovalStrategy = new Function<Collection<Entity>, Entity>() {
         @Override public Entity apply(Collection<Entity> contenders) {
-            // choose newest entity that is stoppable
+            // choose newest entity that is stoppable, or if none are stoppable take the newest non-stoppable
             long newestTime = 0;
             Entity newest = null;
 
             for (Entity contender : contenders) {
-                if (contender instanceof Startable && contender.getCreationTime() > newestTime) {
+                boolean newer = contender.getCreationTime() > newestTime;
```

Comment:
Is there a unit test for this, using the `DynamicCluster` with non-startable entities?